be/src/util/stack_trace_mutex.cpp

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <array>
```

Comment:
Is this necessary?

Reply:
> Is this necessary?

I met a compile error in my test env, this file uses std::array but there is no explicit include, I think add this include is reasonable